releasenotes/notes/deprecate-pulse-package-07a621be1db7fa30.yaml

```
@@ -0,0 +1,80 @@
+---
+deprecations:
+  - |
+    The Qiskit Pulse package is being deprecated and will be removed in Qiskit version 2.0.0. Pulse-level
+    access is currently supported only on a subset of Eagle devices and not supported on the Heron architecture
+    ones. Furthermore, newer IBM Quantum architectures will not support pulse-level access as well.
+    As a consequence, supporting Pulse as a first-class citizen frontend in the Qiskit SDK itself makes little
```

These are mostly nitpicks:

```suggestion
    The Qiskit Pulse package is being deprecated and will be removed in Qiskit 2.0.0. Pulse-level
    access is currently only supported on a subset of Eagle devices and not supported on the Heron architecture. Furthermore, newer IBM Quantum architectures will not support pulse-level access.
    As a consequence, supporting Pulse as a first-class citizen frontend in the Qiskit SDK itself makes little
```

---

Thanks @ElePT for looking at this! A lot of files to review and I indeed contemplated at some point whether it makes sense to split this to smaller PRs. But since the vast majority of changes are trivial, I ended up leaving this as is, hoping it's manageable (and helpful) to review this as a whole. 
Anyway, suggestion looks good.